Mod Menu Helper/assets/fo/lang/en_us.json

```diff
@@ -42,7 +42,7 @@
   "modmenu.summaryTranslation.mixin-conflict-helper": "Alerts when two mods conflict\n§8✎ (no options)",
   "modmenu.summaryTranslation.mixintrace": "Helps developers find the cause of crashes\n§8✎ (no options)",
   "modmenu.summaryTranslation.modmenu": "This mod menu\n§9✎§8 Button placement, filtering, sorting, ...",
-  "modmenu.summaryTranslation.modelfix": "§c*§r Fix transparent lines in held items\n§8✎ (no options)",
+  "modmenu.summaryTranslation.modelfix": "§c*§r Fix transparent lines in all models\n§8✎ (no options)",
```

Comment:
```suggestion
  "modmenu.summaryTranslation.modelfix": "§c*§r Fix transparent lines in all models\n§8✎ (only advanced options)",
```

Reply:
I didn't think of that but it's a good idea